src/Discord.Net.WebSocket/SocketVoiceServer.cs

```diff
@@ -0,0 +1,19 @@
+using System.Diagnostics;
+
+namespace Discord.WebSocket
+{
+    [DebuggerDisplay(@"{DebuggerDisplay,nq}")]
```

Comment:
You need to implement a function named `DebuggerDisplay` for attribute to do its job. See [SocketRole](https://github.com/RogueException/Discord.Net/blob/dev/src/Discord.Net.WebSocket/Entities/Roles/SocketRole.cs#L58) for an example.